source/tutorials/packaging-existing-software.md

```diff
@@ -179,7 +214,7 @@ building
 Great news: the derivation built successfully!
 
 The console output shows that `configure` was called, which produced a `Makefile` that was then used to build the project.
-It wasn't necessary to write any build instructions in this case because the `stdenv` build system is based on `autoconf`, which automatically detected the structure of the project directory.
+It wasn't necessary to write any build instructions in this case because the `stdenv` build system is based on [GNU Autoconf](https://www.gnu.org/software/autoconf/), which automatically detected the structure of the project directory.
```

Comment:
I think this sentence implies `autoconf` itself detected the structure of the project, which is incorrect.

Reply:
What would you suggest here?